plugins/inputs/docker/README.md

```diff
@@ -20,14 +20,22 @@ for the stat structure can be found
   ##   To use TCP, set endpoint = "tcp://[ip]:[port]"
   ##   To use environment variables (ie, docker-machine), set endpoint = "ENV"
   endpoint = "unix:///var/run/docker.sock"
+
   ## Only collect metrics for these containers, collect all if empty
   container_names = []
+
+  ## Containers to include and exclude. Globs accepted.
+  ## Note that the options are ignored when using the container_names option. An empty array for both will include all containers
```

Comment:
We should favor the new options, maybe you can just append `container_names` to the new value?  Then document that the old version is deprecated.  Include the version it is deprecated in and the replacement, something like: "Deprecated (1.4.0), use container_name_include"